ci/environment.yml

```diff
@@ -2,13 +2,13 @@ name: uxarray_build
 channels:
   - conda-forge
 dependencies:
-  - dask
-  - netcdf4
+  - xarray
   - numba
+  - dask
   - numpy
+  - scipy
+  - netcdf4
   - pathlib
   - pre_commit
   - pytest
   - pytest-cov
-  - scipy
-  - xarray
```

Comment:
@philipc2 this file has been in alphabetical order; that's not vital, but we would still not want to break it unless this new order you are making has a functional effect.

Reply:
I'll do a quick re-run in alphabetical order, but it wasn't until I changed the order to have `xarray` `numba` and `dask` at the top of the list that the conda environment built properly.

Let me look into this

Reply:
Appears to be working in alphabetical order, should be good now!